api/recompress_chunk.md

```diff
@@ -1,6 +1,6 @@
 ---
 api_name: recompress_chunk()
-excerpt: Recompress a chunk that had new data inserted after compression
+excerpt: Recompress a chunk to apply new compression settings
```

Comment:
This is not correct, that is not what recompress_chunk does. This procedure is also deprecated. It will call compress_chunk without recompress option cause the previous meaning for recompress was to merge uncompressed data into the compressed chunk and compress_chunk will now always do that.